src/ServicePulse.Host/vue/src/composables/serviceRedirects.js

```diff
@@ -37,15 +37,15 @@ async function getRedirects() {
 export async function useRetryPendingMessagesForQueue(queueName) {
   try {
     const response = await usePostToServiceControl("errors/queues/" + queueName + "/retry");
-    var result = {
+    const result = {
       message: response.ok ? "success" : "error:" + response.statusText,
       status: response.status,
       statusText: response.statusText,
     };
     return result;
   } catch (err) {
     console.log(err);
-    var result_1 = {
+    const result_1 = {
```

Comment:
i suspect this was from Mike using autoconvert to async await: can we just have this object returned rather than having a machine generated variable name?